### PR TITLE
Fix types of arguments in onsite calibration script

### PR DESCRIPTION
**Pull request by maxnoe**
Giving run numbers explicitly did not work due to the wrong type of the arguments

### CODE REVIEW

File: lstchain/scripts/onsite/onsite_create_calibration_file.py [hunk 1]
```diff
@@ -56,7 +54,7 @@ def none_or_str(value):
 optional.add_argument('--sub_run', help="sub-run to be processed.", type=int, default=0)
 optional.add_argument('--min_ff', help="Min FF intensity cut in ADC.", type=float)
 optional.add_argument('--max_ff', help="Max FF intensity cut in ADC.", type=float)
-optional.add_argument('-f','--filters', help="Calibox filters", type=none_or_str)
+optional.add_argument('-f','--filters', help="Calibox filters")
```
Comment (rlopezcoto) [inline lstchain/scripts/onsite/onsite_create_calibration_file.py:57]: no `type` given?

**Comment by FrancaCassol** [comment]
I am a bit surprised, I am using since months these scripts. 
Which arguments did you give?

**Comment by maxnoe** [comment]
I gave explicit run numbers for time and pedestal run. The code did not convert them to int but they were used in the format string with `{:05d}`, so the exception "Invalid format for type str" was raised. You can try it on master, but the types are just wrong for the arguments. For the code to work, the run numbers have to be converted to integers.

File: lstchain/scripts/onsite/onsite_create_calibration_file.py (hunk 1 above)
Comment (maxnoe) [reply lstchain/scripts/onsite/onsite_create_calibration_file.py:57]: Yes, not needed. Filters is a string.

**Comment by FrancaCassol** [comment]
Indeed, I though this part was corrected some time ago. Too many branches probably.

**Review by FrancaCassol — APPROVED** [review]

**Pull request merged**